src/utils.ts

```diff
@@ -0,0 +1,22 @@
+export const getErrorMessage = (error?: string | Error, disabled?: boolean) => {
+  if (!error || disabled) return "";
+  if (error instanceof Error) return error.message;
+  return error;
+}
+
+export const copyText = (text: string): void => {
+  navigator.clipboard.writeText(text).catch(function (err) {
+    console.error('Could not copy text: ', err);
+  });
+};
+
+export const getAbbreviatedAddress = (address: string): string => (
+  `${address.substring(0, 12)}...${address.substring(address.length - 5, address.length)}`
```

Comment:
12 - too much i think

`${address.substring(0, 6)} ... ${address.substring(address.length - 4, address.length)}`

Reply:
i try it. but i see that result if input width = 350px. I think we can pass more information. 
![image](https://user-images.githubusercontent.com/116518132/217831231-8b1baf0f-5fef-4534-8c0d-5c55a8f69a1b.png)

Reply:
Fine
Let's use your choice
And in the "custom styles support" ticket, we need to support custom address formatting